server/src/main/java/org/elasticsearch/index/mapper/CompletionFieldMapper.java

```diff
@@ -313,6 +300,11 @@ public CompletionFieldType fieldType() {
         return (CompletionFieldType) super.fieldType();
     }
 
+    @Override
+    public PostingsFormat postingsFormat() {
+        return PostingsFormat.forName("Completion84");
```

Comment:
Note: PostingsFormat.forName() does the same lazy-instantiation that the previous impl did